includes/utilities.inc

```diff
@@ -183,6 +183,11 @@ function islandora_basic_collection_get_other_parents(AbstractObject $object, Ab
  *   The collection object to add to.
  */
 function islandora_basic_collection_add_to_collection(AbstractObject $new_member, AbstractObject $collection) {
+  // Create hook_islandora_basic_collection_share_migrate()
+  foreach (module_implements('islandora_basic_collection_share_migrate') as $module) {
+    $function = $module . '_islandora_basic_collection_share_migrate';
+    $function($new_member, $collection);
+  }
```

Comment:
So I'm going to have to try this (as I can't keep stuff in my head) but I think these 4 lines could be replaced with
```
module_invoke_all('islandora_basic_collection_share_migrate', $new_member, $collection);
```
https://api.drupal.org/api/drupal/includes%21module.inc/function/module_invoke_all/7.x

Reply:
Addressed with latest commit.